frontend/packages/dev-console/integration-tests/features/topology/topology-chart_area_visual.feature

```diff
@@ -69,37 +77,41 @@ Scenario: Visual for knative service with revisions : T-02-TC03
 
 @regression, @smoke
 Scenario: Context menu of nodes : T-06-TC10
-   Given user is in the topology
+   Given user is at the Topolgy page
    And topology has workloads
```

Comment:
```suggestion
   And user has created workloads
```

Can you please update the other scenarios as well

Reply:
Removed such lines